src/containerapp/azext_containerapp/_params.py

```diff
@@ -49,6 +49,40 @@ def load_arguments(self, _):
     with self.argument_context('containerapp env', arg_group='Virtual Network') as c:
         c.argument('infrastructure_resource_group', options_list=['--infrastructure-resource-group', '-i'], help='Name for resource group that will contain infrastructure resources. If not provided, a resource group name will be generated.', is_preview=True)
 
+    # App Resiliency
+    with self.argument_context('containerapp resiliency') as c:
+        c.argument('resource_group_name', arg_type=resource_group_name_type, id_part=None)
+        c.argument('container_app_name', options_list=['--container-app-name'], help=f"The name of the Container App. A name must consist of lower case alphanumeric characters or '-', start with a letter, end with an alphanumeric character, cannot have '--', and must be less than {MAXIMUM_CONTAINER_APP_NAME_LENGTH} characters.")
+        c.argument('name', name_type, help=f"The name of the Container App Resiliency Policy. A name must consist of lower case alphanumeric characters or '-', start with a letter, end with an alphanumeric character, cannot have '--', and must be less than {MAXIMUM_APP_RESILIENCY_NAME_LENGTH} characters.")
+        c.argument('yaml', type=file_type, help='Path to a .yaml file with the configuration of a container app resiliency policy. All other parameters will be ignored.')
+        c.argument('default', options_list=['--default'], help='Set recommended values of resiliency policies for a container app.')
+
+    with self.argument_context('containerapp resiliency', arg_group='Timeout Policy') as c:
+        c.argument('timeout_response_in_seconds', type=int, options_list=['--timeout'], help='Specifies the timeout in seconds. This spans between the point at which the entire request has been processed and when the response has been completely processed. This timeout includes all retries. Default: 30.')
+        c.argument('timeout_connection_in_seconds', type=int, options_list=['--timeout-connect'], help='The timeout in seconds for new network connections to the container app. Default: 5.')
+
+    with self.argument_context('containerapp resiliency', arg_group='HTTP Retry Policy') as c:
+        c.argument('http_retry_max', type=int, options_list=['--http-retry'], help='Specifies the allowed number of retries. Default: 3.')
+        c.argument('http_retry_delay_in_milliseconds', type=int, options_list=['--http-delay'], help='Specifies the base interval between retries. Default: 1000.')
```

Comment:
All descriptions of arguments should be original form

Reply:
I took it from here: 
https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/route/v3/route_components.proto#config-route-v3-retrypolicy-retrybackoff